### PR TITLE
Added a note regarding the application.conf `db.connectionTimeout`.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
@mcovarr first reviewer

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -135,6 +135,7 @@ database {
     hsqldb {
       db.url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
       db.driver = "org.hsqldb.jdbcDriver"
+      db.connectionTimeout = 1000 // NOTE: The default 1000ms is often too short for production mysql use
```
Comment (mcovarr) [inline src/main/resources/application.conf]: should these not be 1000 then?

Comment (mcovarr) [inline src/main/resources/application.conf]: or maybe not have the comment for hsqldb instances?

**Comment by kshakir** [comment]
The wheel chose @geoffjentry as a second reviewer, but I'm happy to have anyone else add their 2¢

**Comment by cjllanwarne** [comment]
:+1: interestingly this was already set to 5000 in the vault ctmpl file.

**Comment by kshakir** [comment]
Yep, GotC isn't using vault. Good 'ol file system + vi :wink:

**Comment by kshakir** [comment]
Going to squash d212ae1 and bafb678 as I think this is close to mergable.

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**